annotation-error-decoder/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
```

Comment:
I saw in multiple other modules that there were build.gradle files. I assumed (perhaps a mistake) that there was potentially a migration underway, and added the file to help out. I'm absolutely fine to delete.